collections.d/christmas.yml

```diff
@@ -38,8 +39,8 @@ Christmas:
   - The Polar Express
   - The Ref
   - The Santa Clause
+  - The Sound of Music
   - Trading Places
   - Where the .+? is Santa
   - White Christmas
   - ^Die Hard$
-  - The Family Stone
```

Comment:
It looks like this was removed. By accident?

Reply:
it was in the list twice.